src/dotnet/commands/dotnet-tool/install/ProjectRestorer.cs

```diff
@@ -47,6 +50,11 @@ public ProjectRestorer(IReporter reporter = null)
 
             argsToPassToRestore.Add($"-verbosity:{verbosity ?? "quiet"}");
 
+            if (_alwaysForwardArguments != null && _alwaysForwardArguments.Any())
```

Comment:
Nit: is the `Any` check necessary?  I assume `AddRange` handles a sequence of length 0.

Reply:
fixed, removed any check